tips/TIP-0032/tip-0032.md

```diff
@@ -0,0 +1,104 @@
+| Below Max Depth                       | 15               | dynamic | uint8  | Below Max Depth is the maximum allowed delta value between Oldest Cone Root Index (OCRI) of a given block in relation to the current Confirmed Milestone Index (CMI) before it gets lazy. |
+|                                       |                  |         |        |                                                                                                                                                                                           |
+| Max Shimmer Genesis Supply            | 1450896407249092 | static  | uint64 | Total amount of Shimmer genesis supply denominated in `glow`. 1 `glow` = 0.000001 `SMR`                                                                                                   |
+| Max Inputs Count                      | 128              | dynamic | uint32 | Maximum number of inputs in a transaction payload.                                                                                                                                        |
+| Max Outputs Count                     | 128              | dynamic | uint32 | Maximum number of outputs in a transaction payload.                                                                                                                                       |
+| Max Native Token Count                | 64               | dynamic | uint32 | Maximum number of different native tokens that can be referenced in one transaction.                                                                                                      |
+|                                       |                  |         |        |                                                                                                                                                                                           |
+| Max Tag Length                        | 64               | dynamic | uint8  | Maximum length of a `Tag` field in bytes.                                                                                                                                                 |
+| Max Metadata Length                   | 8192             | dynamic | uint16 | Maximum length of a `Metadata` field in bytes.                                                                                                                                            |
+|                                       |                  |         |        |                                                                                                                                                                                           |
+| VByte Cost                            | 500              | dynamic | uint32 | Minimum amount of Shimmer (denominated in `glow`) that need to be deposited per vbyte of an output.                                                                                       |
```

Comment:
This doesn't seem to correspond with https://github.com/iotaledger/hornet/issues/1584#issuecomment-1178767669